src/executioners/project_configuration.cpp

```diff
@@ -36,9 +39,16 @@ bool project_configuration::set_active_scene(unsigned int new_scene) {
 		return false;
 	}
 	const auto new_scene_index = this->scene_id_mapping.at(new_scene);
-	this->scenes[this->get_active_scene()].on_stop();
+	if (const auto current_scene = this->get_active_scene(); current_scene < this->scenes.size()) {
+		this->scenes[current_scene].on_stop();
+	}
 	this->current_active_scene = new_scene_index;
-	this->scenes[this->get_active_scene()].on_start();
+	if (new_scene_index >= this->scenes.size()) {
+		this->current_active_scene = this->scenes.size() - 1;
+	}
```

Comment:
what happens here? just to make sure that the index is in the range? this could not lead to en endless loop, cause it cant switch scenes automatically

Reply:
The idea was that it revers to an id that is still in range. Regarding the loop: this worked yesterday. How would the loop look like?

Reply:
just if the scene would automatically call the next one until its number is scenes.size() its gets reset to size()-1 and again wants to call the next one...